docs/jobs-queue/queues.mdx

```diff
@@ -25,7 +25,32 @@ Then, you could configure two different runner strategies:
 
 ## Executing jobs
 
-As mentioned above, you can queue jobs, but the jobs won't run unless a worker picks up your jobs and runs them. This can be done in two ways:
+As mentioned above, you can queue jobs, but the jobs won't run unless a worker picks up your jobs and runs them. This can be done in three ways:
+
+#### Cron jobs
+
+You can use the jobs.autoRun property to configure cron jobs:
+
+```ts
+export default buildConfig({
+  // Other configurations...
+  jobs: {
+    tasks: [
+      // your tasks here
+    ],
+    autoRun: (payload) => [
+      {
+        default: '*/5 * * * *', // every hour at minute 0
```

Comment:
Is this property supposed to be `cron`, not `default`?

Reply:
hm, I had tested it by moving it to _community config... I would have expected to get "Object literal may only specify known properties". There must be something we could do better with types.